### ISSUE TITLE
build fail of base/iconv.cc, FreeBSD

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/mozc#27]
```
version: mozc-0.13.464.102
target file: base/iconv.cc
category: build fail
OS: FreeBSD 9-CURRENT / 8.1-RELEASE
CC: gcc 4.2.1 (FreeBSD default compiler)


base/iconv.cc build fails on FreeBSD because of follow line:

     if (iconv(ic, reinterpret_cast<char **>(&ibuf), &ilen, &obuf, &olen)

follow line is acceptable.

     if (iconv(ic, (const char **)(&ibuf), &ilen, &obuf, &olen)

Please change that line or fix it in another way.
```

Original issue reported on code.google.com by `daichig...@gmail.com` on 20 Sep 2010 at 2:58

**Comment by yukawa** [comment]
Just FYI, this is indeed an issue in FreeBSD side.  See [FreeBSD PR 199099](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=199099), which was recently fixed.

**Comment by yukawa** [comment]
Reopening as this can be fixed in conjunction with #252.
